rudder-web/src/main/scala/com/normation/rudder/appconfig/ConfigService.scala

```diff
@@ -62,6 +62,7 @@ import com.normation.rudder.domain.eventlog.ModifyAgentRunStartMinuteEventType
 import com.normation.rudder.domain.eventlog.ModifyAgentRunSplaytimeEventType
 import com.normation.rudder.reports._
 import com.normation.rudder.domain.eventlog.ModifyRudderSyslogProtocolEventType
+import scala.language.implicitConversions
```

Comment:
In 2.11, it raises compiler warning without it, because it is "an advanced feature". Add the import let the compiler you didn't wrote "implicit" by mistake, I guess